src/import/planner.c

```diff
@@ -38,7 +38,7 @@
 #include "compat/compat.h"
 #include "planner.h"
 
-#if PG12
+#if PG12 || (PG13 && PG_VERSION_NUM < 130003)
```

Comment:
Nit: PG13 check seems superfluous:

```suggestion
#if PG12 || (PG_VERSION_NUM < 130003)
```